src/antsibull/data/ansible-readme.rst

```diff
@@ -90,14 +119,12 @@ License
 
 GNU General Public License v3.0 or later
 
-See `COPYING <COPYING>`_ to see the full text.
+See `COPYING <COPYING>`_ for the full license text.
```

Comment:
This link is dead. Not sure where it should point to.

Reply:
That's a good question. https://github.com/ansible/ansible/blob/devel/COPYING maybe? The corresponding file in antsibull that's included in Ansible releases (https://github.com/ansible-community/antsibull/blob/main/src/antsibull/data/gplv3.txt) is also a possibility.

Reply:
> The corresponding file in antsibull that's included in Ansible releases (https://github.com/ansible-community/antsibull/blob/main/src/antsibull/data/gplv3.txt) is also a possibility.

I think that's the best option

Reply:
https://github.com/ansible-community/antsibull/pull/474/commits/ffa27a7d7abf5ccdb14be2646f729157581926af